test/README.md

```diff
@@ -167,6 +201,52 @@ Use the following commands to initialize the server and run a Smoke Test:
 Note that any previously installed RackHD Vagrant boxes will prevent a new instance from running.
 Please remove any old RackHD VMs prior to executing this routine.
 
+## Hints and background for logging/debuging tests
+If you are not familar with the logging system in general, please read through 'stream_monitor/flogging/README.md'
+
+A cheat sheet of key information from that readme includes:
+* There are loggers for four main areas of code: test.run, test.data, infra.run, and infra.data.
+    * Each of these defaults to DEBUG (meaning DEBUG_1 through DEBUG_9 are _not_ captured by default).
+    * Each has a log file associated with it, named test_run.log, test_data.log and so on. (see below for location of output logs)
+    * Each also contributes to a combined_all_all.log.
+* There is a root logger that has these properties:
+    * INFO level as a gate.
+    * Handler to output to stdout.
+    * A handler with a log file named console_capture.log.
+
+The logfiles are generated per-run in 'log_output/run_<timedate-stamp>.d/' under
+this (RackHD/test) directory. In addition, 'log_output/run_last.d' is linked to
+the last test-run.
+
+The following are some common use-cases for wanting to alter logging for some reason:
```

Comment:
To me, these examples should be in a separate readme or in the logging readme above.  We don't dig into any of the command line options in this top level RackHD/readme.md.